hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -50,6 +50,23 @@
 
 CORS_ALLOW_CREDENTIALS = True
 
+RECAPTCHA_DOMAIN = "www.recaptcha.net"
+
+RECAPTCHA_TEST_PUBLIC_KEY = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
+RECAPTCHA_TEST_PRIVATE_KEY = "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
+
+if DEBUG:
```

Comment:
My bad, I forgot debug is false during testing. Change this to `if DEBUG or IN_TESTING` and all should be well